cli/src/render.rs

```diff
@@ -268,9 +270,11 @@ impl Renderable for PackageType {
 
 impl Renderable for PackageStatusExtended {
     fn render(&self) -> String {
+        let time = Local.timestamp((self.basic_status.last_updated / 1000) as i64, 0);
```

Comment:
From the docs:

> Panics on the out-of-range number of seconds and/or invalid nanosecond, for a non-panicking version see [timestamp_opt](https://docs.rs/chrono/latest/chrono/offset/trait.TimeZone.html#method.timestamp_opt).

I'm not sure what values are "out-of-range" in that context, but it's probably better for us to print `"UNKNOWN"` or something if that ever happens rather than to panic.

Reply:
Seems highly unlikely unless something goes seriously wrong in the backend, but I guess it doesn't hurt.